### PR TITLE
[#98024644] Approvals::Individual and a single approval chain entry point

### PR DESCRIPTION
**Pull request by cmc333333**
Split out from #454 

This doesn't change too much of `app` - it migrates all approvals to the `Approvals::Individual` subclass and removes all but one entry point for setting the approval chain.

Most of the code updates are to the specs, which were heavy users of `add_approver`

### CODE REVIEW

File: app/models/approvals/individual.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      end
+      state :actionable do
+        event :approve, transitions_to: :approved
+      end
+      state :approved
+    end
+
+    def self.for_email(email)
+      Approvals::Individual.new(user: User.for_email(email))
+    end
```
Comment (afeld) [inline app/models/approvals/individual.rb]: `User#for_email` will create the record if it doesn't exist yet...thinking we want to be consistent about the behavior for both, or at least make a note that they are different. Doesn't look like this method is being used, though?

File: spec/features/display_status_spec.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 describe "Display status text" do
   let(:proposal) { FactoryGirl.create(:proposal, :with_parallel_approvers) }
   before do
+    proposal.approvers.first.update(first_name: "Uniquely", last_name: "Named")
+    proposal.approvers.second.update(first_name: "Onlyof", last_name: "Itskind")
```
Comment (afeld) [inline spec/features/display_status_spec.rb]: Is this necessary?

**Comment by afeld** [comment]
Just a couple small things – otherwise, looks good!

File: app/models/approvals/individual.rb (hunk 1 above)
Comment (cmc333333) [inline app/models/approvals/individual.rb]: Easy enough to get rid of now and bring it back under a different name if it's needed later

File: spec/features/display_status_spec.rb (hunk 2 above)
Comment (cmc333333) [inline spec/features/display_status_spec.rb]: Yes (the "lino" name wasn't too unique); I'll update the relevant tests to make it more apparent how this is used

**Comment by cmc333333** [comment]
Updated

**Pull request merged**